assembl/tests/views_tests/test_graphql.py

```diff
@@ -104,7 +104,15 @@ def test_mutation_create_thematic_with_video(graphql_request):
                 u'title': u'Understanding the dynamics and issues',
                 u'identifier': 'survey',
                 u'video': {u'title': u"Laurent Alexandre, chirurgien et expert en intelligence artificielle nous livre ses prédictions pour le 21e siècle.",
+                           u'titleEntries': [{
```

Comment:
Ahh never mind. The csv export is created from the fixtures of thematics, sentiments, etc...